### ISSUE TITLE
请问oneflow中的反向传播和梯度累积是怎么做的

### ISSUE THREAD

**Issue by ximu1211** [Oneflow-Inc/oneflow#3860]
例如pytorch中一般会是
        optimizer.zero_grad()
        # forward + backward + optimize
        outputs = net(inputs)
        loss = criterion(outputs, labels)
        loss.backward()
        optimizer.step()
想知道oneflow中是否有类似的操作，或者oneflow在这方面有没有不同的地方

**Comment by daquexian** [comment]
你好！在 oneflow 里，反向传播相关参数是通过 https://docs.oneflow.org/basics_topics/optimizer_in_function_config.html#_5 这里描述的方式：在 job 函数内部调用 optimizer 的 minimize 方法来设置的。

用户调用 job 函数时，oneflow 会自动完成反向传播和权重更新。不需要手动执行 `optimizer.zero_grad()`, `loss.backward()` 和 `optimizer.step()`。

关于梯度累积，我们即将推出原生的 mini batch 功能（https://github.com/Oneflow-Inc/oneflow/pull/3734 ），敬请期待
